pages/pipelines/defining_steps.md.erb

```diff
@@ -71,20 +73,40 @@ steps:
       queue: 'deploy'
 ```
 
-See the [Buildkite Agent pipelines documentation](/docs/agent/cli-pipeline) for a full list of step types and supported options.
+## Step types
 
-## Dynamic pipelines
+There are four different step types you can use in your Buildkite pipelines:
```

Comment:
Do you think it's worth saying something like 'click through for more information on each'?